src/install-node.ts

```diff
@@ -25,7 +25,14 @@ export function generateNodeTarballUrl(
 	} else {
 		ext = 'tar.gz';
 	}
-	return `https://nodejs.org/dist/${version}/node-${version}-${plat}-${arch}.${ext}`;
+
+	let mirror = process.env.ZEIT_FUN_MIRRORS_NODE;
```

Comment:
I would prefer the name to be:

```suggestion
	let mirror = process.env.ZEIT_FUN_NODE_MIRROR;
```